src/content/translations/pt-br/roadmap/beacon-chain/index.md

```diff
@@ -15,15 +15,15 @@ summaryPoint3: A Beacon Chain introduziu a lógica de consenso e o protocolo de
 
 ## O que era a Beacon Chain? {#what-is-the-beacon-chain}
 
-Beacon Chain foi o nome original dado a cadeia de blocos de prova de participação lançada em 2020. Ela foi criada para garantir que a lógica de consenso de prova de participação era sólida e sustentável antes de implementá-la na rede principal do Ethereum. Portanto, ela existia paralelamente à prova de trabalho do Ethereum. Desligar a prova de trabalho e mudar para a prova de participação no Ethereum exigiu instruir a Beacon Chain a aceitar transações da cadeia original do Ethereum, juntá-las em blocos e então organizá-las em uma cadeia de blocos usando um mecanismo de consenso baseado em prova de participação. Ao mesmo tempo, os nós da rede original do Ethereum desligaram sua lógica de consenso, propagação de blocos e mineração, passando essas funções para a Beacon Chain. Este evento ficou conhecido como [A Fusão](/roadmap/merge/). Assim que A Fusão aconteceu, não haviam mais duas cadeias de blocos, mas apenas uma cadeia de prova de participação do Ethereum.
+Beacon Chain foi o nome original dado a cadeia de blocos de prova de participação lançada em 2020. Ela foi criada para garantir que a lógica de consenso de prova de participação era sólida e sustentável antes de implementá-la na rede principal do Ethereum. Portanto, ela existia paralelamente à prova de trabalho do Ethereum. Desligar a prova de trabalho e mudar para a prova de participação no Ethereum exigiu instruir a Beacon Chain a aceitar transações da cadeia original do Ethereum, juntá-las em blocos e então organizá-las em uma cadeia de blocos usando um mecanismo de consenso baseado em prova de participação. Ao mesmo tempo, os nós da rede original do Ethereum desligaram sua lógica de consenso, propagação de blocos e mineração, passando essas funções para a Beacon Chain. Este evento ficou conhecido como [The Merge](/roadmap/merge/). Assim que A Fusão aconteceu, não haviam mais duas cadeias de blocos, mas apenas uma cadeia de prova de participação do Ethereum.
```

Comment:
Not sure why they're using the english term here ('The Merge' instead of the previous translation)

Reply:
Not sure, but this should be handled in crowdin